README.md

```diff
@@ -3,10 +3,15 @@
 
 Our goal is to create a survey that helps us, data science & MLOps practitioners, learn more about our community and (in the future) track how our community changes year-over-year.  Said differently - this is a survey for the community, by the community.  We designed this survey to gain an accurate understanding of we, the professionals designing, building and deploying **production** ML systems, individually and collectively are, the types of data and models we implement, the tools we use, and the challenges we face.  Specifically, this is NOT a vendor-led survey that pushes forward a specific perspective that advantages a vendor or group of vendors.
 
-The anonymized responses (e.g., identifying information removed) will be made available as part of this repo.
-
 This document provides an outline of the topics we want to understand from our community.  It is intended to spark discussion and drive alignment on set of topics and learning goals within each topic.
 
+# Objectives & goals
+
+Specifically, our goals are:
+* Develop a ~15 minute survey that provides a neutral, vendor-agnostic view of how practitioners apply data science
```

Comment:
is that really what we care about? i thought it was more interesting to know about the ML setups or how they are doing MLOps? 

> how practitioners apply data science

Reply:
I was trying to summarize all of that as “apply data science”

Maybe “implement data science in the real world”?